lib/node_modules/@stdlib/plot/components/svg/symbols/lib/main.js

```diff
@@ -17,14 +17,16 @@
 */
 
 'use strict';
-
+// eslint-disable-next-line no-warning-comments
```

Comment:
```suggestion

```

Reply:
We want to flag warning comments such as this. Disabling prevents us from doing so. That also applies below. Please just focus on updating the loop.